isUnique/isUnique.js

```diff
@@ -3,9 +3,30 @@
 * Extra Credit - Answer this question - What if you cannot use additional data structures?
 */
 
+// Create a loop to check individual characters and check to see if it was used before
+// charAt() method returns the character at a specific index.
+// str.indexOf('a', 5) find the first occurance of a letter in a string. ex: a, start position at 5
 const isUnique = (str) => {
```

Comment:
Using nested loops can get heavy, but this is a very common solution to this problem. ( no need to respond back to me with my next question ) Could you think of a way to do this without nesting forLoops?